examples/non-transaction-retry/non_transaction_retry.out

```diff
@@ -3,4 +3,7 @@
 ballerina run non_transaction_retry.bal
 Attempting execution...
 Attempting execution...
-Work done.
+Work is done.
+Attempting execution...
+Attempting execution...
+Work is done.
```

Comment:
```suggestion
Work completed..
```
Why is this line repeated? Saying "Work is done/completed is a bit ambiguous. Can we specify more details about what was done?